aws_cloudtrail_rules/aws_console_login_without_mfa.py

```diff
@@ -10,6 +14,14 @@ def rule(event):
     session_context = deep_get(event, "userIdentity", "sessionContext", default={})
     response_elements = event.get("responseElements", {})
 
+    # If Account is less than 3 days old do not alert
+    if isinstance(check_new_user(deep_get(event, "userIdentity", "userName")), str):
+        if ast.literal_eval(
+            check_new_user(deep_get(event, "userIdentity", "userName"))
+        ):
+            logging.debug("check_new_user is a mocked string for unit testing")
+            return False
```

Comment:
I think we might be missing the check for new user when its not a string.  Also, I think we can call it one time rather than two: 
```suggestion
    # If Account is less than 3 days old do not alert
    is_new_user = check_new_user(deep_get(event, "userIdentity", "userName"))
    if isinstance(is_new_user, str):
        logging.debug("check_new_user is a mocked string for unit testing")
        is_new_user = ast.literal_eval(is_new_user)
    if is_new_user:
        return False
```